bool ok
---
geometry_msgs/Point waypoint
bool new_waypoint